scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -159,6 +161,21 @@ namespace <%= namespace %> {
         }
 
         /// <summary>
+        /// Launches an external browser with the Cart's web checkout link loaded. This can be used to perform
+        /// a cart checkout when no native web view is available, such as on desktop, or from the Unity editor.
+        /// </summary>
+        /// <param name="success">called when the game has regained focus and the checkout was successful.</param>
+        /// <param name="cancelled">called when the game has regained focus but the checkout has not completed.</param>
+        /// <param name="failure">called when the game has regained focused and an error has occured during checkout.</param>
+        public void CheckoutWithExternalBrowser(CheckoutSuccessCallback success, CheckoutCancelCallback cancelled, CheckoutFailureCallback failure) {
```

Comment:
Good points, I forgot the reason why @mikkoh preferred opt 2, but now that we are aligning external web browsers and web views, it seems like opt 3 provides ease of use for the implementor